packages/integration-tests/src/helper.ts

```diff
@@ -61,8 +61,12 @@ export class Helper {
     return new Promise((resolve) => setTimeout(resolve, ms));
   }
 
-  async performGETrequest(apiRoute: string) {
-    return request(environment.blockExplorerAPI).get(apiRoute);
+  async performGETrequest(apiRoute: string, network = "local") {
```

Comment:
Could we make the 2nd argument is optional or is local - the default value for all other cases?